pandas/tests/io/test_pytables.py

```diff
@@ -4540,6 +4540,19 @@ def test_pytables_native2_read(self, datapath):
             d1 = store['detector']
             assert isinstance(d1, DataFrame)
 
+    def test_legacy_table_fixed_format_read_py2(self, datapath):
+        # legacy table with fixed format written en Python 2
+        with ensure_clean_store(
+                datapath('io', 'data', 'legacy_hdf', 'legacy_table_fixed_py2.h5'),
+                mode='r') as store:
+            with catch_warnings():
+                simplefilter("ignore", pd.io.pytables.IncompatibilityWarning)
```

Comment:
what does this come up?

Reply:
Copy / past error, this filter is useless

Reply:
Removed